base/abstractarray.jl

```diff
@@ -3,9 +3,10 @@
 ## Basic functions ##
 
 """
-    AbstractArray{T, N}
+    AbstractArray{T,N}
 
-Abstract array supertype which arrays inherit from.
+Abstract `N`-dimensional array supertype with elements of type `T`
+which arrays inherit from.
```

Comment:
I know this is how it was before, but personally I find this wording quite awkward. Perhaps it would be worth clarifying it a bit while you're in here? Maybe something like
> Abstract `N`-dimensional array supertype containing elements of type `T`. `Array`s and some array-like objects inherit from this type.

? Up to you if you want to change it right now though.